documentation/modules/oauth/con-oauth-authentication-broker.adoc

```diff
@@ -41,121 +29,234 @@ spec:
         type: internal
         tls: true
         authentication:
-          type: *oauth*
+          type: oauth
+      - name: external3
+        port: 9094
+        type: loadbalancer
+        tls: true
+        authentication:
+          type: oauth    
       #...
 ----
 
-You can configure OAuth 2.0 authentication in your listeners.
-We recommend using OAuth 2.0 authentication together with TLS encryption (`tls: true`).
-Without encryption, the connection is vulnerable to network eavesdropping and unauthorized access through token theft.
-
-You configure an `external` listener with `type: oauth` for a secure transport layer to communicate with the client.
-
-.Using OAuth 2.0 with an external listener
-[source,yaml,subs="+quotes"]
+.Enabling SASL authentication mechanisms
+
+Use one or both of the following SASL mechanisms for clients to exchange credentials and establish authenticated sessions with Kafka.
+
+`OAUTHBEARER`:: Using the `OAUTHBEARER` authentication mechanism, credentials exchange uses a bearer token provided by an OAuth callback handler. 
+Token provision can be configured to use the following:
++
+--
+* Client ID and secret (using the the OAuth 2.0 _client credentials mechanism_)
+* Long-lived access token obtained manually
```

Comment:
The `obtained manually` is no longer the only option as the access token can be made available by some external mechanism and mounted into a pod as a file. An example is the Service Account token file when using Kubernetes API server as authorization server.
```suggestion
* Long-lived access token
```